deepfence_frontend/apps/dashboard/src/features/topology/components/node-details/SummaryTables.tsx

```diff
@@ -0,0 +1,209 @@
+import { useEffect, useState } from 'react';
+import { twMerge } from 'tailwind-merge';
+
+import { ModelContainer, ModelContainerImage, ModelProcess } from '@/api/generated';
+
+const tableClass = 'w-full table-fixed';
+const tableHeaderClass = 'text-gray-600 dark:text-gray-400';
+const tableBodyClass = 'text-gray-700 dark:text-gray-300';
+const tableHeaderCellClass = 'text-sm font-semibold text-left';
+const tableBodyCellClass = 'text-sm font-normal text-left';
+const tableRowClass = 'hover:bg-gray-100 dark:hover:bg-gray-700 cursor-pointer';
+const moreButtonClass =
+  'text-sm font-semibold text-gray-500 dark:text-gray-400 uppercase w-full text-right';
+
+const SUMMARY_ROWS = 10;
+
+export const ProcessTable = ({ processes }: { processes: ModelProcess[] }) => {
+  const [sortedData, setSortedData] = useState(processes);
+  const [showAll, setShowAll] = useState(false);
+
+  const first5Rows = sortedData.slice(0, SUMMARY_ROWS);
+  const remainingRows = sortedData.slice(SUMMARY_ROWS).length;
+
+  useEffect(() => {
+    setSortedData(processes);
+  }, [processes]);
+
+  return (
+    <div>
```

Comment:
Are you ok, if this table is not component table(look and feel)?

Reply:
yes, we don't want component's table look and feel here, component's tables are huge and wont fit properly in this sidebar.